### DISCUSSION TITLE
game doesn't close supposedly /pressing back button (android)

### DISCUSSION THREAD

**Discussion by Fxhsrx** [ppy/osu#27560] [Q&A] [answered]
This is just a silly thing i found while messing with settings, nothing really game breaking;
(The solution is to just close the game manually, nothing bad happens)

When opengl is the current selected renderer then you change it to automatic, the game asks you to close the game then launch it again but it just gets stuck after the closing sequence is done playing. 

https://github.com/ppy/osu/assets/116128416/41896981-3497-4992-bff9-d6e3a76a4910

An actual question: 
Pressing back on the game standby (main menu) screen doesn't close the game but just basically minimizes it.
Is this supposed to actually close (stop the game) / play the closing sequence first then close the game or that's intentional?

https://github.com/ppy/osu/assets/116128416/bd338d70-7198-493a-8fd6-a851bd726202

**Comment by nagi-desuuu** [comment, marked as answer]
https://github.com/ppy/osu/discussions/16546

**Reply by Fxhsrx** [reply]
thanks for the context, too bad we won't be seeing the closing sequence when backing.
